certification/engine/engine.go

```diff
@@ -99,12 +99,9 @@ var (
 		viper.GetString("pyxis_api_token"),
 		viper.GetString("certification_project_id"),
 		&http.Client{Timeout: 60 * time.Second}))
-	// runnableContainerCheck  certification.Check = containerpol.NewRunnableContainerCheck(internal.NewPodmanEngine())
```

Comment:
Did we already remove the implementations for these? Or are you thinking that will be in another PR?